sdk/core/Azure.Core/tests/TransportFunctionalTests.cs

```diff
@@ -938,7 +938,11 @@ private async Task CanCancelContentUpload(bool https)
                 async context =>
                 {
                     // read part of the request
+#if NET6_0_OR_GREATER
+                    await context.Request.Body.ReadExactlyAsync(buffer, 0, 100);
+#else
                     await context.Request.Body.ReadAsync(buffer, 0, 100);
```

Comment:
[CA2022: Avoid inexact read with Stream.Read](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2022) ReadAsync may return fewer bytes than requested which can result in unreliable code. ReadExactlyAsync should be used instead but it's only available for .NET6+. Since this is a test case, just leave the net462 version as-is.